lib/board.rb

```diff
@@ -17,19 +17,16 @@ def reset
     @grid = Array.new(@rows) { Array.new(@columns, CELL) }
   end
 
-  def print_board
+  def board_with_axis
     x_axis = [*1..@columns].map { |n| "#{n} " }
     grid_with_x_axis = [*@grid, x_axis]
 
-    grid_with_x_and_y_axis = grid_with_x_axis.map.with_index do |row, index|
+    grid_with_x_axis.map.with_index do |row, index|
       if index == @rows
         [' ', *row].join(' ')
       else
         [@rows - index, *row].join(' ')
```

Comment:
interestingly `' '` is a concept that is **"only"** related to our `Display` being a terminal display! for example, this would not work with HTML as all spaces are equal to 1 space. As such you will need to devise a smarter way of doing this. Maybe as far as the board goes, it is only a 2-dimensional array, but there is a `DisplayBoard` class that is extended by a `DisplayBoard::Terminal` class that is specific to using characters that are specific to a board printed in a terminal?